src/ontology/imports/foodon_product_import.owl

```diff
@@ -33368,6 +33368,52 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004142 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004142">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00002951"/>
+        <obo:IAO_0000115 xml:lang="en">A patty made from rabbit meat.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date xml:lang="en">2021-12-20T15:39:21Z</dc:date>
+        <rdfs:label xml:lang="en">rabbit patty</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004143 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004143">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00004125"/>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date xml:lang="en">2021-12-20T15:36:23Z</dc:date>
+        <rdfs:label xml:lang="en">braided chew</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004144 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004144">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001283"/>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-12-20T16:09:45Z</dc:date>
+        <rdfs:label xml:lang="en">poultry by-product</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004145 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004145">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00002239"/>
+        <obo:IAO_0000115 xml:lang="en">A lengthwise incision on the front or back of the shrimp in a symmetrical way.</obo:IAO_0000115>
```

Comment:
A type of processed shrimp in which a lengthwise incision on the front or back of the shrimp  has been done to allow easier extraction of shrimp meat.